package.json

```diff
@@ -10,9 +10,10 @@
     "@types/jest": "^26.0.14",
     "jest": "^26.4.2",
     "marked": "^0.4.0",
-    "prettier": "^2.1.2",
+    "prettier": "^2.7.1",
     "ts-jest": "^26.4.1",
     "ts-node": "^8.4.1",
+    "ts-toolbelt": "^9.6.0",
```

Comment:
Could also just copy over the `Narrow` type if you'd rather not add a dep.

Reply:
yeah let's copy the type please. It would need to be a runtime dependency otherwise, right?

Reply:
I can do that. It's not really a _runtime_ dependency in the sense that no code from `ts-toolbelt` will be downloaded in the browser (its only type helpers, no actual code). But the library's type definitions will include the TS toolbelt type.

Reply:
I think there is one or two real runtime functions in ts-toolbelt, which is why it has [129 bytes](https://bundlephobia.com/package/ts-toolbelt@9.6.0). It can probably be tree-shaken away if we don't use it, but having "zero dependencies" is also something that sells nicely 🤷